clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/helpers/EnableDisablePercentCategorizer.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.helpers
+
+
+class EnableDisablePercentCategorizer<T> {
+  /**
+   * During an enable/disable operation that accepts a desired percentage of instances to leave enabled/disabled, this acts
+   * as a helper function to return which instances still need to be enabled/disabled.
+   *
+   * @param modified are the instances that don't need to be enabled/disabled (presumably have already been enabled/disabled).
+   * @param unmodified are the instances that do need to be enabled/disabled.
+   * @param desiredPercent is the end desired percent. If the percent has already been achieved or exceeded by the input instances, we return an empty list.
```

Comment:
Seems like the second sentence in this @param description should be merged into the @return description.